<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConstraintManagerWidget</class>
 <widget class="QWidget" name="ConstraintManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Constraint Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <property name="spacing">
    <number>10</number>
   </property>
   <item>
    <widget class="QGroupBox" name="selectionGroup">
     <property name="title">
      <string>Selection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="infoLabel">
        <property name="text">
         <string>Select Child then Parent</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="selectionList"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="refreshButton">
          <property name="text">
           <string>Refresh List</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setParentButton">
          <property name="text">
           <string>Set as Parent</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setChildButton">
          <property name="text">
           <string>Set as Child</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearSelectionButton">
          <property name="text">
           <string>Clear Selection</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="constraintGroup">
     <property name="title">
      <string>Create Constraint</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="typeLabel">
        <property name="text">
         <string>Constraint Type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="constraintTypeCombo">
        <item>
         <property name="text">
          <string>Parent/Child</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Position</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rotation</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Aim</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Relation</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="activeLabel">
        <property name="text">
         <string>Active:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="activeCheckbox">
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="snapButton">
        <property name="text">
         <string>Snap</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
